plugins/ipam/dhcp/lease.go

```diff
@@ -344,6 +360,45 @@ func (l *DHCPLease) maintain() {
 	}
 }
 
+func (l *DHCPLease) checkLinkExistsWithBackoff(ctx context.Context) (bool, error) {
+	checkFunc := func() (bool, error) {
+		_, err := netlink.LinkByName(l.link.Attrs().Name)
+		if err != nil {
+			var linkNotFoundErr *netlink.LinkNotFoundError = &netlink.LinkNotFoundError{}
+			if errors.As(err, linkNotFoundErr) {
```

Comment:
Owe you one @zeeke !!! Thanks. Appreciate that, it works a charm and much cleaner than the string check, thanks.